news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2017-11-05
+--- 
+
+![](filepath) BANNER IN PROGRESS
+
+After a long hiatus, Chaos, Voli and I are happy to announce the revival of official beatmapping contests: Welcome to the _osu! Mapping Olympiad!_
+
+Let me elaborate on the concept for the osu! Mapping Olympiad. To ensure that there is a healthy and consistent flow of official contests, 
+we have worked on a concept which involved all types of modes - three contests per mode will happen annually in a specific order. 
+To keep things interesting, we have also worked on different kind of contests - so stay tuned in the upcoming months!
+
+### Songs (STILL NEED HELP WITH THE SONG PREVIEWS)
+
+To start off, we are launching this contest with not one, but **three** songs for you to tackle! Showcasing a variety of our Featured Artists, have yourself a listen to the tracks you’ll be taking on:
+
+- Kuba Oms - Beautiful Uncertainty (Redux)
+- OISHII - Honey Snowflakes
+- cySmix - Escapism
+
+There’s going to be something for everyone’s taste - funky rhythms by Kuba Oms, fast-paced goodness by OISHII and some chill beats by cYsmix!
+
+### Ruleset
+
+- This contest is for the **osu!** (standard) mode only.
+- Each entry should follow the Ranking Criteria.
+- Beatmap submissions may contain files up to 15 MB.
+- Storyboarding will not be considered in the judging.
+- You may submit **one** difficulty for **one** song. Multiple submissions for multiple songs will not be accepted.
+- You may collaborate if you like, but only the submitter of each entry will receive credit for judging and prizes.
```

Comment:
Can be shortened to "Collaboration is allowed, but only the submitting mapper is eligible for any prizes"

Does contribution matter? Could i just submit a diff i have placed 2 objects on and be judged as the guy that takes credit? Seems like massive abyss of possible exploits

Reply:
Again taken from previous contest rules. I suppose we can simply put a rule that your entries have to be your own mapping, would like to discuss this a bit I think.